consent/strategy_default.go

```diff
@@ -1155,6 +1182,76 @@ func (s *DefaultStrategy) HandleOAuth2AuthorizationRequest(
 	return consentSession, f, nil
 }
 
+// HandleOAuth2DeviceAuthorizationRequest handles the device authorization flow
+func (s *DefaultStrategy) HandleOAuth2DeviceAuthorizationRequest(
+	ctx context.Context,
+	w http.ResponseWriter,
+	r *http.Request,
+) (_ *flow.AcceptOAuth2ConsentRequest, _ *flow.Flow, err error) {
+	ctx, span := trace.SpanFromContext(ctx).TracerProvider().Tracer("").Start(ctx, "DefaultStrategy.HandleOAuth2DeviceAuthorizationRequest")
+	defer otelx.End(span, &err)
+
+	deviceVerifier := strings.TrimSpace(r.URL.Query().Get("device_verifier"))
+	loginVerifier := strings.TrimSpace(r.URL.Query().Get("login_verifier"))
+	consentVerifier := strings.TrimSpace(r.URL.Query().Get("consent_verifier"))
```

Comment:
use GetRequestForm

Reply:
`r` is not a fosite `Requester`. The device verification endpoint is not part of the RFC so it wasn't defined in fosite.